docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -3,8 +3,8 @@ title: Using Teleport Connect
 description: Using Teleport Connect
 ---
 
-Teleport Connect provides easy and secure access to SSH servers, databases, and Kubernetes clusters,
-with support for other resources coming in the future.
+Teleport Connect provides easy and secure access to SSH servers, databases, and Kubernetes clusters
+as an intuitive desktop GUI application, with support for other resources coming in the future.
```

Comment:
Intuitive is subjective. Let's rephrase this. Maybe something like

_Teleport Connect is a desktop app that provides secure access ..._